plugins/pvcmount.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: pvcmount
+spec:
+  version: "v1.0.0"
+  shortDescription: Go binary for mounting persistent volumes.
+  homepage: https://github.com/Stenstromen/kubectl-pvcmount
+  description: |
+    Go binary for temporary mounting of PVCs to a pod. 
+    This is useful for debugging purposes or for copying data from a PVC to another location.
+    The pvc is mounted to /mnt in a pod with a busybox container.
+    Examples:
+    `$ kubectl pvcmount pvc -p pvc-name -n namespace-containing-pvc`
```

Comment:
I recommend not listing usage here. 

Anyone curious will run -h/-help or check your readme. This thing here will only go out of date.